pandas/tests/extension/test_sparse.py

```diff
@@ -402,6 +402,16 @@ class TestPrinting(BaseSparseTests, base.BasePrintingTests):
     def test_array_repr(self, data, size):
         super().test_array_repr(data, size)
 
+    def test_fillna_repr(self):
```

Comment:
Can you put this test in one of the files in `tests/arrays/sparse/..` (eg in test_dtype.py)? 

(trying to keep those extension tests *only* the shared common ones inherted from the base tests, any custom test goes into `tests/arrays` instead of `tests/extension/`)